test/common/init/manager_impl_test.cc

```diff
@@ -22,9 +22,11 @@ TEST(InitManagerImplTest, AddImmediateTargetsWhenUninitialized) {
 
   ExpectableTargetImpl t1("t1");
   m.add(t1);
+  m.checkUnreadyTargets();
```

Comment:
I don't think that adding these checks in the unit tests does anything. That is, the unit test isn't testing the content of the debug messages, right?

Reply:
Yes, agreed. I will remove them in the next commit.